optuna/study.py

```diff
@@ -223,7 +223,7 @@ def optimize(
         """
 
         if not self._optimize_lock.acquire(False):
-            raise RuntimeError("Nested invocations of `Study.optimize` method aren't allowed.")
+            raise RuntimeError("Nested invocation of `Study.optimize` method aren't allowed.")
```

Comment:
Please replace `aren't` with `isn't`.

Reply:
Sorry for the careless mistake. Fixed: https://github.com/pfnet/optuna/pull/491/commits/c64f0fb51cb85457d73bd8e00b0f8006a364e717